shared/storage/aws.py

```diff
@@ -147,7 +147,7 @@ def delete_file(self, bucket_name, path):
         except ClientError:
             raise
 
-    def delete_files(self, bucket_name, paths=[]):
```

Comment:
This default value is dangerous and unnecessary. fixed the occurrences I found across the project.